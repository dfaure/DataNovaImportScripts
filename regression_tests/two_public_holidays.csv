00285A;LEHAUCOURT AP;2020-11-24;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-02-11;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-10;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-03;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-08;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-01;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-11;15:00-18:00;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-02-16;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-15;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-16;08:30-11:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-02;08:30-11:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-02-18;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-20;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-04;15:00-18:00;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-11-21;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-11-27;15:00-18:00;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-11-23;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-11-22;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-07;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-06;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-11-20;15:00-18:00;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-09;08:30-11:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-11-26;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-12;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-11-16;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-14;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-11-17;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-11-28;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-11-19;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-07;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-18;15:00-18:00;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-14;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-15;15:00-18:00;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-22;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-12;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-06;08:30-11:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-13;08:30-11:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-16;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-19;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-26;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-11;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-22;15:00-18:00;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-27;08:30-11:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-04;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-02-07;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-01;15:00-18:00;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-02-05;15:00-18:00;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-02-03;08:30-11:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-08;15:00-18:00;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-02-01;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-11-25;08:30-11:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-10;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-09;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-17;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-25;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-11-29;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-11-30;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-24;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-27;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-23;08:30-11:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-05;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-05;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-19;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-03;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-23;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-18;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-02-09;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-13;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-20;08:30-11:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-17;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-26;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-28;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-02-06;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-25;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-30;08:30-11:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-31;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-21;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-31;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-02;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-24;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-02-04;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-21;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-02-08;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-29;15:00-18:00;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-02-02;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-28;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-01-30;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-12-29;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-02-13;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-02-12;15:00-18:00;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-02-15;13:30-16:30;;;;;;;15:00;15:00;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2020-11-18;08:30-11:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-02-14;FERME;;;;;;;;;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-02-17;08:30-11:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
00285A;LEHAUCOURT AP;2021-02-10;08:30-11:30;;;;;;;11:30;11:30;;;FERME;;;;;;;
