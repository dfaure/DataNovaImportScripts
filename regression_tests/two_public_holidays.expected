00285A|LEHAUCOURT AP|Mo-Tu,Th 13:30-16:30; We 08:30-11:30; Fr 15:00-18:00; PH off; 2021 Jan 01 15:00-18:00
